Modules/_zstd/decompressor.c

```diff
@@ -638,7 +533,7 @@ stream_decompress(ZstdDecompressor *self, Py_buffer *data, Py_ssize_t max_length
 
 error:
     /* Reset decompressor's states/session */
-    decompressor_reset_session(self, type);
+    decompressor_reset_session(self);
 
     Py_CLEAR(ret);
 success:
```

Comment:
Perhaps remove the `success` label and just duplicate the `return ret;` statements?